.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 4,
+    "semi": true,
+    "printWidth": 80,
+    "singleQuote": true
+}
```

Comment:
Add lint and lint:fix to scripts in `packages.json`

```js
    "lint": "eslint \"**/*.ts\" --ignore-pattern \"**/*.d.ts\" --quiet",
    "lint:fix": "eslint \"**/*.ts\" --ignore-pattern \"**/*.d.ts\" --fix --quiet"
```

Add `.eslintrc` file. See cdk-template [.eslintrc](https://github.com/magiqsoftware/cdk-template/blob/main/.eslintrc.js).

Can then run `npm run lint`.

Reply:
done.